### PR TITLE
fixed error when transition is not configured

### PR DESCRIPTION
**Pull request by ar4s**

### CODE REVIEW

File: src/ralph/admin/views/extra.py [hunk 1]
```diff
@@ -86,7 +84,7 @@ class RalphListView(RalphExtraViewMixin, RalphTemplateView):
     extra_view_base_template = 'admin/extra_views/base_list.html'
 
     def get_context_data(self, **kwargs):
-        context = super(RalphListView, self).get_context_data(**kwargs)
```
Comment (szok) [inline src/ralph/admin/views/extra.py:89]: :+1:

File: src/ralph/lib/transitions/admin.py [hunk 2]
```diff
@@ -34,11 +35,7 @@ class TransitionAdminMixin(object):
     run_transition_view = RunTransitionView
 
     def get_transition_url_name(self, with_namespace=True):
-        params = self.model._meta.app_label, self.model._meta.model_name
-        url = '{}_{}_transition'.format(*params)
-        if with_namespace:
-            url = 'admin:' + url
-        return url
+        return get_transition_url_name(self.model, with_namespace)
```
Comment (szok) [inline src/ralph/lib/transitions/admin.py]: maybe use: get_model_view_url_name
https://github.com/allegro/ralph/blob/ng/src/ralph/helpers.py

File: src/ralph/admin/views/extra.py [hunk 3]
```diff
@@ -127,15 +125,11 @@ class RalphDetailView(
...
-        )
+        return super().dispatch(request, model, *args, **kwargs)
 
     def get_context_data(self, **kwargs):
-        context = super(RalphDetailView, self).get_context_data(**kwargs)
-        context['object'] = self.object
-        context['original'] = self.object
-        context['change_views'] = self.views
+        context = super().get_context_data(**kwargs)
+        context['object'] = context['original'] = self.object
```
Comment (szok) [inline src/ralph/admin/views/extra.py:133]: :+1:

**Comment by szok** [comment]
:ok:

File: src/ralph/admin/views/extra.py [hunk 4]
```diff
@@ -127,15 +126,11 @@ class RalphDetailView(
...
-        return super(RalphDetailView, self).dispatch(
-            request, model, *args, **kwargs
-        )
+        return super().dispatch(request, model, *args, **kwargs)
 
     def get_context_data(self, **kwargs):
-        context = super(RalphDetailView, self).get_context_data(**kwargs)
-        context['object'] = self.object
-        context['original'] = self.object
-        context['change_views'] = self.views
```
Comment (mkurek) [inline src/ralph/admin/views/extra.py:138]: isn't it necessary?

File: src/ralph/lib/transitions/admin.py [hunk 5]
```diff
@@ -51,6 +48,7 @@ def change_view(self, request, object_id, form_url='', extra_context=None):
         )
 
     def get_urls(self):
+        print('get', self)  # DETELE THIS
```
Comment (mkurek) [inline src/ralph/lib/transitions/admin.py]: so delete this :)

File: src/ralph/settings/test.py [hunk 6]
```diff
@@ -10,8 +10,6 @@
     }
 }
 
-PASSWORD_HASHERS = ('django_plainpasswordhasher.PlainPasswordHasher', )
```
Comment (mkurek) [inline src/ralph/settings/test.py:13]: what about it?

Comment (ar4s) [inline src/ralph/settings/test.py:13]: `PASSWORD_HASHERS` option was duplicated in `test.py` scope

**Comment by mkurek** [comment]
:ok:

**Pull request merged**